packages/eslint-config-generator/src/config-generator.test.ts

```diff
@@ -63,7 +63,8 @@ describe('Configuration generator', function () {
               '**/postcss.config.{js,ts}',
               '**/tailwind.config.{js,ts}',
               '**/vite.config.{js,ts}',
-              '**/prettier.config.{js,ts}',
+              '**/prettier.config.{js,cjs,ts}',
+              '**/.prettierrc.{js,cjs,ts}',
```

Comment:
In fact it looks like Prettier does not support `.ts` files, see https://prettier.io/docs/en/configuration.html

So you can remove `,ts` for both entries, thanks.

Reply:
done